ddtrace/tracer/tracer.go

```diff
@@ -682,6 +682,7 @@ func spanStart(operationName string, options ...StartSpanOption) *Span {
 		traceprof.SetProfilerRootTags(span)
 	}
 	if isRootSpan || context.span.service != span.service {
+		// The span is the local root span.
```

Comment:
Is this and the comment on line 755 both supposed to say *local* root span?

Reply:
Yes, because line 686 sets the metric used later in line 754 identifying the span as the local root.